src/utils/dev.js

```diff
@@ -34,7 +34,7 @@ const ENV_VAR_SOURCES = {
     printFn: chalk.red,
   },
   ui: {
-    name: 'build settings',
+    name: 'site settings',
```

Comment:
Per [this conversation](https://netlify.slack.com/archives/C02TE7L1QUA/p1658512126111859)